addons/website/static/src/scss/primary_variables.scss

```diff
@@ -1980,18 +1980,18 @@ $o-base-website-values-palette: (
     'btn-ripple': false,
 
     'btn-padding-y': null, // Default to BS
-    'btn-padding-x': null, // Default to BS
+    'btn-padding-x': 1rem,
```

Comment:
I'm wondering if we should create a specific `$o-website-values-palettes` map for these default options to be active for the default theme **only**. 
The "risk" is that themes that don't define buttons' designs (and thus rely on default), may inherit these custom value instead of the BS default ones.

Not sure if it's a problem, to check with the devs.